gateway/src/apicast/http_proxy.lua

```diff
@@ -81,6 +83,20 @@ local function absolute_url(uri)
     )
 end
 
+function fsize (path)
+  local file, err = open(path)
```

Comment:
This is I/O blocking, I don't think we want to do this on every request as it will really hurt performance.